.github/workflows/airflow-operator.yml

```diff
@@ -26,7 +26,7 @@ on:
       - 'client/python/**'
       - 'build/python-client/**'
       - 'pkg/api/*.proto'
-      - '.github/workflows/python.yml'
+      - '.github/workflows/python-client.yml'
       - 'docs/python_armada_client.md'
```

Comment:
We should add this to the list but it isnt that important. 
      - '.github.com/workflows/python-tests/action.yml'

Reply:
done